docs/view_components/welcome.rst

```diff
@@ -0,0 +1,38 @@
+linking ************
+Welcome page
+************
+
+A welcome window can be seen in almost every software/website in production. The welcome window should serve as an orientation point for the user and let them know, in as brief words (and other audiovisual content) as possible, what to expect from the tool. It is also the launch point from where the user begins their energy systems simulation.
```

Comment:
The first sentence is a general statement which does not bring much to the description

Reply:
Removed now